system/Database/BaseConnection.php

```diff
@@ -194,7 +195,14 @@ abstract class BaseConnection implements ConnectionInterface
      * @var array
      */
     protected $failover = [];
-
+    
+    /**
+     * Application name for Postgres DSN.
+     *
+     * @var array
+     */
+    protected $application_name = [];
```

Comment:
This property is only for Postgre.
Don't add unrelated property in BaseConnection.
Move it to the Postgre class.